exercises/Exercises.All.sln

```diff
@@ -0,0 +1,1502 @@
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "SimpleLinkedList", "simple-linked-list\SimpleLinkedList.fsproj", "{DAABE08C-95DD-404F-9B66-0645E410A867}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "SpaceAge", "space-age\SpaceAge.fsproj", "{FA64EA2B-91B3-446D-92B9-90FD6803FE93}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Strain", "strain\Strain.fsproj", "{948968C5-BFA9-417C-A549-1E95D9197C6B}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Sublist", "sublist\Sublist.fsproj", "{4544EC02-2E1F-49BC-A7BC-329BAB929077}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "SumOfMultiples", "sum-of-multiples\SumOfMultiples.fsproj", "{AAD0D1BD-C3E3-40C1-A074-25FBD4693885}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Tournament", "tournament\Tournament.fsproj", "{4EEC333A-3AE7-41E8-A11E-F41B48FFD3FB}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Transpose", "transpose\Transpose.fsproj", "{3E9C6B89-1BE6-4959-9E08-0E9BEBB812D5}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "TreeBuilding", "tree-building\TreeBuilding.fsproj", "{0886ED65-D98C-4537-9A0A-566CF4A26C21}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Triangle", "triangle\Triangle.fsproj", "{5238BED1-9448-457A-9E9C-A238B12B387A}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "TwelveDays", "twelve-days\TwelveDays.fsproj", "{530E2EEA-6087-4E06-8155-CAC9367DCE33}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "TwoBucket", "two-bucket\TwoBucket.fsproj", "{E39689D7-539F-43F5-AF8E-AEA130FB37F1}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "TwoFer", "two-fer\TwoFer.fsproj", "{FBE3DED0-5997-4A70-B2DB-5CCD6678F303}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "VariableLengthQuantity", "variable-length-quantity\VariableLengthQuantity.fsproj", "{F14CA20D-F85E-4F9F-951E-E2465F5429C5}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "WordCount", "word-count\WordCount.fsproj", "{C8E73F9B-C894-4AEA-BEAA-C1D2FE35246B}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "WordSearch", "word-search\WordSearch.fsproj", "{55EB1DF1-2760-494B-A5BC-A733A906A91F}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Wordy", "wordy\Wordy.fsproj", "{B14FF411-3EDE-4E2F-98AB-78CBDFFBAF48}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "ZebraPuzzle", "zebra-puzzle\ZebraPuzzle.fsproj", "{3C57692F-7E65-4C6F-9F2D-3962A9007EE2}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Zipper", "zipper\Zipper.fsproj", "{1DEE9AFA-D88A-4143-B53D-E494FF331FC8}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
```

Comment:
Minor: we don't need `x64` or `x86` builds the `Any CPU` is enough

Reply:
Good point!